linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/wrong_value.fail.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
-# platform = Red Hat Enterprise Linux 7,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_wrlinux
```

Comment:
Here there is also a different test scenario specific for RHEL8 so this one shouldn't be executed on RHEL8.